src/lib/Microsoft.Health.Fhir.R4.Ingest/Service/R4DeviceAndPatientCreateIdentityService.cs

```diff
@@ -10,10 +10,13 @@
 using Microsoft.Health.Extensions.Fhir;
 using Microsoft.Health.Extensions.Fhir.Service;
 using Microsoft.Health.Fhir.Ingest.Data;
+using Microsoft.Health.Fhir.Ingest.Host;
 using Model = Hl7.Fhir.Model;
 
 namespace Microsoft.Health.Fhir.Ingest.Service
 {
+    [ResourceIdentityService("Create")]
+    [ResourceIdentityService("R4DeviceAndPatientCreateIdentityService")]

```

Comment:
Better to have this as [ResourceIdentityService(nameof(R4DeviceAndPatientCreateIdentityService))] to avoid typos in the string.

Reply:
Updated.